src/test/common/extensionMessaging.test.ts

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
-import {HostPlatform} from "../../common/hostPlatform";
-
 import {
     ExtensionMessage,
-    ExtensionMessageSender,
+    MessagingChannel,
 } from "../../common/extensionMessaging";
 
+import {
```

Comment:
I would keep the import statements consistent with the other files, using one line only.